### ISSUE TITLE
Boolean values in env, when default is set to true, always yield true

### ISSUE THREAD

**Issue by pwntr** [thenativeweb/processenv#2]
Once setting the an env variable to `false` with a default when `undefined` of `true` (adding `|| true`):
```javascript
const dryrun = processenv('DRYRUN') || true;
console.log(dryrun);

const dryrun_direct = process.env.DRYRUN;
console.log(dryrun_direct);
```
The result becomes:
```
true
false
```
No env, env set to `false` or `true` all yield the same first result for variable `dryrun`.

**Comment by pwntr** [comment]
fixed with #3

**Comment by goloroden** [comment]
Thanks for your PR, it is now live on npm as `1.1.0` 😊
